MagTag_Christmas_Countdown/code.py.py

```diff
@@ -0,0 +1,121 @@
+import time
+import board
+import displayio
+from adafruit_magtag.magtag import MagTag
+import adafruit_il0373
+from adafruit_display_shapes.circle import Circle
+
+#  create MagTag and connect to network
+magtag = MagTag()
+magtag.network.connect()
+
+#  setup display for use with displayio
+displayio.release_displays()
+display_bus = displayio.FourWire(board.SPI(), command=board.EPD_DC,
+                                 chip_select=board.EPD_CS,
+                                 reset=board.EPD_RESET, baudrate=1000000)
+time.sleep(1)
+display = adafruit_il0373.IL0373(
+    display_bus,
+    width=296,
+    height=128,
+    rotation=270,
+    black_bits_inverted=False,
+    color_bits_inverted=False,
+    grayscale=True,
+    refresh_time=1,
+    seconds_per_frame=1
+)
+
+#  displayio group
+group = displayio.Group(max_size=30)
+
+#  import tree bitmap
+tree = displayio.OnDiskBitmap(open("/atree.bmp", "rb"))
+
+tree_grid = displayio.TileGrid(tree, pixel_shader=displayio.ColorConverter())
+
+#  circles to cover-up bitmap's number ornaments
+c_1 = Circle(246, 53, 11, fill = 0xFF00FF)
```

Comment:
can you turn this into a list of postiions and then iterate to create an array of Circles instead of doing em one by one